stage-2/get-version-from-pr/action.yml

```diff
@@ -21,3 +22,22 @@ runs:
           exit 1
         fi
         echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+    - name: Install utils
+      shell: bash
+      run: python3 -m pip install -q packaging
+
+    - name: Get is prerelease?
+      id: prerelease
+      shell: python
+      run: |
+        import os
+        from packaging import version
+
+        ver = version.Version(os.environ['VERSION'])
+        is_pre = str(ver.is_prerelease).lower()
```

Comment:
From the [`packaging`](https://packaging.pypa.io/_/downloads/en/latest/pdf/) docs

>is_prereleaseA boolean value indicating whether this Version instance represents a prerelease and/or development release.

I believe it follows what's defined in [PEP-440](https://www.python.org/dev/peps/pep-0440/#pre-releases) for pre-releases:

```
X.YaN   # Alpha release
X.YbN   # Beta release
X.YrcN  # Release Candidate
X.Y     # Final release
```

So alpha, beta, and release candidates will be treated as pre-releases👍